docs: difference from jest in snapshots with custom messages (#2863)
vitest-dev/vitest: docs/guide/snapshot.md

  }
})
```

> #### 3. Chevron `>` is used as a separator instead of colon `:` for hints
> 
> Vitest uses chevron `>` as a separator instead of colon `:` for readability.

Author: ```suggestion
#### 3. Chevron `>` is used as a separator instead of colon `:` for custom messages

Vitest uses chevron `>` as a separator instead of colon `:` for readability, when a custom message is passed during creation of a snapshot file.
```

For the following example test code:
```js

> test("toThrowErrorMatchingSnapshot", () => {

Author: ```suggestion
test('toThrowErrorMatchingSnapshot', () => {
```

  expect(() => {

>     throw new Error("error");

Author: ```suggestion
    throw new Error('error')
```

>   }).toThrowErrorMatchingSnapshot("hint");

Author: ```suggestion
  }).toThrowErrorMatchingSnapshot('hint')
```

> });

Author: ```suggestion
})
```
